docs/build/js-client/3-hello-world.md

```diff
@@ -121,61 +118,58 @@ Using the code generated by the compiler is as easy as calling a function. The c
 Let's see how to use the generated code in our application. The `index.ts` file looks this way:
 
 ```typescript
-import "@fluencelabs/js-client.node"; // Import the JS Client implementation. Don't forget to add this import!
-import { Fluence } from "@fluencelabs/js-client.api"; // Import the API for JS Client
-import { kras } from "@fluencelabs/fluence-network-environment"; // Import list of possible relay nodes (network environment)
+import { 
+    Fluence, // Import the API for JS Client
+    kras // // Import list of possible relay nodes (network environment)
+} from "@fluencelabs/js-client";  
 import {
   registerHelloWorld,
   sayHello,
   getRelayTime,
   tellFortune,
 } from "./_aqua/hello-world"; // Aqua compiler provides functions which can be directly imported like any normal TypeScript function.
 
-async function main() {
-  await Fluence.connect(kras[3]); // Connecting to the fourth kras node.
+await Fluence.connect(kras[3]); // Connecting to the fourth kras node.
```

Comment:
maybe would be somewhat useful to suggest using a randomKras here. Don't know?

Reply:
agree. Updated